lib/samples/apply_simple_renderer_to_feature_layer/apply_simple_renderer_to_feature_layer_sample.dart

```diff
@@ -85,11 +83,13 @@ class _ApplySimpleRendererToFeatureLayerSampleState
 
   void resetRenderer() {
     _featureLayer.resetRenderer();
```

Comment:
```suggestion
    // reset the renderer on the feature layer
    _featureLayer.resetRenderer();
```
We're also adding in comments throughout the samples, again somewhat verbose in terms of best practice, but supports the workflows of reading through samples to all levels of developers both from a Flutter and ArcGIS perspective. Note the most recent PRs as examples (comments are missing from the original implementations)